tools/telemetry/www/doc/TOC.md

```diff
@@ -0,0 +1,29 @@
+[HTML5 Boilerplate homepage](https://html5boilerplate.com)
```

Comment:
I think we should link to the github repo or some other source rather than package them for end users.